src/array.vala

```diff
@@ -2,6 +2,9 @@ using GLib;
 
 public class Vast.Array : Object
 {
+    public const ssize_t NEW_AXIS = ssize_t.MAX;
+    public const size_t DEFAULT_DIMENSION = size_t.MAX;
```

Comment:
Not its insufficient -- what if I want a new axis in the middle? e.g. `[:, Newaxis, :]`?